### PR TITLE
Fixing the schema for slack_channel_override

### PR DESCRIPTION
**Pull request by JeffAshton**
Fixes schema for slack_channel_override

### CODE REVIEW

File: elastalert/schema.yaml [hunk 1]
```diff
@@ -495,7 +495,7 @@ properties:
   ### Slack
   slack_webhook_url: *arrayOfString
   slack_username_override: {type: string}
-  slack_channel_override: {type: string}
+  slack_channel_override: *arrayOfStrings
```
Comment (JeffAshton) [inline elastalert/schema.yaml:498]: `slack_channel_override` can be either a single string or many:

https://github.com/jertel/elastalert2/blob/c982b9489be8f0856a943e3bc7bd9d532b284701/elastalert/alerters/slack.py#L22

File: elastalert/schema.yaml [hunk 2]
```diff
@@ -495,7 +495,7 @@ properties:
   ### Slack
   slack_webhook_url: *arrayOfString
   slack_username_override: {type: string}
-  slack_channel_override: {type: string}
+  slack_channel_override: *arrayOfString
```
Comment (JeffAshton) [inline elastalert/schema.yaml:498]: `slack_channel_override` can be either a single string or many:

https://github.com/jertel/elastalert2/blob/c982b9489be8f0856a943e3bc7bd9d532b284701/elastalert/alerters/slack.py#L23

**Review by jertel — APPROVED** [review]

**Comment by jertel** [comment]
Thank you!

**Comment by JeffAshton** [comment]
> Thank you!

No, thank you sir for picking up this project.  I was on the fence about doing the same.

**Comment by jlmolinyawe** [comment]
Is it possible to push a new image version with this fix?

**Comment by jertel** [comment]
Yes, there's also a helm change that's waiting on a release. Planning to do one this weekend.

**Comment by jlmolinyawe** [comment]
Thank you!

**Pull request merged**